doc/api/perf_hooks.md

```diff
@@ -98,6 +98,43 @@ added: v8.5.0
 Returns the current high resolution millisecond timestamp, where 0 represents
 the start of the current `node` process.
 
+### performance.startEventLoopTimer([options])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `options` {Object}
+  * `resolution` {number} The sampling rate in milliseconds. Defaults to `10`.
+  * `reset` {boolean} `true` to reset the sampling data. Defaults to `false`.
+* Returns: {boolean} True if the timer was started. False if the timer has
+           already been started.
+
+Starts monitoring the event loop for delays. The accumulated event loop delay
+is reported as `performance.nodeTiming.eventLoopDelay`.
+
+The accumulated event loop delay (`delay`) is determined by:
+
+* Calculating a `threshold` in milliseconds as `5e6 + (resolution * 1e6)`.
```

Comment:
If my unit conversions are right, isn't this an obscure way of saying the threshold is 1.5 times the resolution?

Reply:
Yep